README.md

```diff
@@ -126,6 +126,12 @@ Open VSCode Editor and Press `ctrl+P`, type `ext install live-sass`.
      ``` 
      <hr>
 
+* **`liveSassCompile.settings.watchOnLaunch` :** Set this to `true` if you want Live Sass Compiler to automatically start watching your .sass or .scss file when you open an applicable workspace.
+    * *Default value is `true`*
```

Comment:
Oops! Default is `false`

Reply:
Ah man. What a silly thing to over look :(

Reply:
Actually, I was confused. I was thinking where you made the mistake - in the docs or `project.json`..

Anyway, I've published the #41 to marketplace..  so, this PR will be published by tomorrow..

Thanks a lot for your contribution